en/pd-microservices.md.md

```diff
@@ -0,0 +1,62 @@
+---
+title: PD Microservices
+summary: Learn how to enable the microservice mode of PD to improve service quality.
+---
+
+# PD Microservices
+
+Starting from v8.0.0, PD supports the microservice mode, which disaggregates the timestamp allocation and cluster scheduling functions of PD into the following two independently deployed microservices. In this way, these two functions are decoupled from the routing function of PD, which allows PD to focus on the routing service for metadata.
+
+- TSO microservice: provides monotonically increasing timestamp allocation for the entire cluster.
+- Scheduling microservice: provides scheduling functions for the entire cluster, including but not limited to load balancing, hot spot handling, replica repair, and replica placement.
+
+Each microservice is deployed as an independent process. If you configure more than one replica for a microservice, the microservice automatically implements a primary-secondary fault-tolerant mode to ensure high availability and reliability of the service.
+
+## Usage scenarios
+
+PD microservices are commonly used to address performance bottlenecks in PD and improve PD service quality. With this feature, you can avoid the following issues:
+
+- Long-tail latency or jitter in TSO allocations due to excessive pressure in PD clusters
+- Service unavailability of the entire cluster due to failures in the scheduling module
+- Bottleneck issues solely caused by PD
+
+In addition, when the scheduling module is changed, you can update the scheduling microservice independently without restarting PD, thus avoiding any impact on the overall service of the cluster.
+
+> **Note:**
+>
+> If the performance bottleneck of a cluster is not caused by PD, there is no need to enable microservices, because using microservices increases the number of components and raises operational costs.
+
+## Restrictions
+
+- Currently, the TSO microservice does not support dynamic start and stop. After enabling or disabling the TSO microservice, you need to restart the PD cluster for the changes to take effect.
+- Only the TiDB component supports a direct connection to the TSO microservice through service discovery, while other components need to forward requests to the TSO microservice through PD to obtain timestamps.
+- Microservices are not compatible with the [Data Replication Auto Synchronous](https://docs.pingcap.com/tidb/stable/two-data-centers-in-one-city-deployment) feature.
+- Microservices are not compatible with the TiDB system variable [`tidb_enable_tso_follower_proxy`](https://docs.pingcap.com/tidb/stable/system-variables#tidb_enable_tso_follower_proxy-new-in-v530).
+- Due to the potential presence of hibernate Regions in a cluster, during a primary and secondary switchover of the scheduling microservice, the cluster scheduling function of the cluster might be unavailable for up to five minutes to avoid redundant scheduling.
```

Comment:
```suggestion
- Due to the potential presence of hibernate Regions in a cluster, during a primary and secondary switchover of the scheduling microservice, the scheduling function of the cluster might be unavailable for up to five minutes to avoid redundant scheduling.
```